utils/rpms/daos.spec

```diff
@@ -10,9 +10,11 @@
 %else
 %global daos_build_args client test
 %endif
-%global mercury_version   2.4
+%global mercury_version   2.4.0-7%{?dist}
 %global libfabric_version 1.15.1-1
-%global argobots_version 1.2
+%global argobots_version 1.2-1%{?dist}
+%global fuse_version 3.16.2-2%{?dist}
+%global spdk_version 22.01.2-6%{?dist}
 %global __python %{__python3}
```

Comment:
Can we simplify `pmdk_version` handling?
```
%if (0%{?suse_version} >= 1500)
%global pmdk_version 2.1.0-3.suse1500
%else
%global pmdk_version 2.1.0-3%{?dist}
%endif
```
and use `%{pmdk_version}` in the rest of the file.